es6-shim.js

```diff
@@ -1370,32 +1364,29 @@
     },
 
     sign: function sign(value) {
-      var number = Number(value);
-      if (number === 0) { return number; }
-      if (Number.isNaN(number)) { return number; }
-      return number < 0 ? -1 : 1;
+      var x = Number(value);
+      if (x < 0) {
+        return -1;
+      }
+      if (x > 0) {
+        return 1;
+      }
+      return x;
     },
 
     sinh: function sinh(value) {
       var x = Number(value);
-      if (!globalIsFinite(x) || x === 0) { return x; }
-
       if (Math.abs(x) < 1) {
         return (Math.expm1(x) - Math.expm1(-x)) / 2;
       }
-      return (Math.exp(x - 1) - Math.exp(-x - 1)) * Math.E / 2;
+      return (Math.exp(x - 1) - Math.exp(-x - 1)) * (Math.E / 2);
```

Comment:
mathematically - not, but from floating point perspective - should, because of overflow, also "(Math.E / 2)" in brackets allows js engines to precalculate constant value during, I think